src/components/ItemCard.css

```diff
@@ -0,0 +1,15 @@
+.Item-card {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  background-color: var(--color-items-background);
+  margin: 1rem;
+  border: hidden;
```

Comment:
border: hidden means, the border is hidden :P.. but the place is beeing reserved.
if you dont want that the space of the border is reserved, you can instead use "none"

Reply:
Thank you for the clarification. I read somewhere that border: none and border:hidden would have the same function but I do not have any special plan for the border at this stage.